packages/cloudfront-signer/src/sign.spec.ts

```diff
@@ -604,3 +604,49 @@ describe("getSignedCookies", () => {
     expect(verifySignature(denormalizeBase64(result["CloudFront-Signature"]), policy)).toBeTruthy();
   });
 });
+
+describe("getSignedUrl- when signing a URL with a date range", () => {
+  const dateString = "2024-05-17";
```

Comment:
Use a precise date down to the second in tests. Millisecond number or ISO date string with timezone.

This YYYY-MM-DD date string only works because of the specific JS runtime, and I don't believe is generally spec-compatible with the Date constructor.